otx/algorithms/action/tasks/inference.py

```diff
@@ -296,6 +295,14 @@ def _get_metric(self, output_resultset):
             return MetricsHelper.compute_f_measure(output_resultset)
         raise NotImplementedError(f"{self._task_type} is not supported in action task")
 
+    def remove_empty_frames(self, dataset):
```

Comment:
Some functions still remain without typehint.

Reply:
Given that restricted time, I think all function would be impossible, but I'll add typehint as much as I can.
And I think local test for action task is enough for this PR, what do you think?

Reply:
> Given that restricted time, I think all function would be impossible, but I'll add typehint as much as I can. And I think local test for action task is enough for this PR, what do you think?

Agree with your opinion, we need to continuously refactor for readability. 
Local test for action tasks (cls, det) is enough + pre-commit tests

Reply:
Cool, I wll update local test results after adding type hint.